src/App.js

```diff
@@ -1,16 +1,26 @@
-import React from 'react';
+import React from "react";
+
+let todoList = [
+  {
+    id: "1",
+    title: "Complete React assignment",
+  },
+  { id: "2", title: "Pick up lunch" },
+  {
+    id: "3",
+    title: "Play Zelda",
+  },
+];
 
 function App() {
   return (
-    <div style={{ textAlign: 'center' }}>
-      <header>
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a href="https://reactjs.org" target="_blank" rel="noopener noreferrer">
-          Learn React
-        </a>
-      </header>
+    <div>
+      <h1>Todo List</h1>
+      <ul>
+        {todoList.map(function (item) {
+          return <li key={item.id}>{item.title}</li>;
+        })}
```

Comment:
Great job , just nitty suggestion, we can use more descriptive name for parameters something like todoItem, also check out using arrow function with map function something like this,

`{todoList.map((todoItem) => <li key={todoItem.id}>{todoItem.title}</li>)} `

here is refence doc for map function https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map

Reply:
Got it, will do that - thanks!!